src/ppx/application.cpp

```diff
@@ -674,9 +670,10 @@ void Application::DispatchConfig()
         AddAssetDir(path / "third_party/assets");
     }
 
-    if (!mStandardOptions.assets_paths.empty()) {
+    auto assetPaths = mStandardOpts.pAssetsPaths->GetValue();
+    if (!assetPaths.empty()) {
         // Insert at front, in reverse order, so we respect the command line ordering for priority.
-        for (auto it = mStandardOptions.assets_paths.rbegin(); it < mStandardOptions.assets_paths.rend(); it++) {
+        for (auto it = assetPaths.rbegin(); it < assetPaths.rend(); it++) {
```

Comment:
options are parsed in the correct order no?
So `rbegin` should be `begin`.
(was probably wrong before..)